backend/.env.sample

```diff
@@ -2,3 +2,7 @@
 DATABASE_URL=postgres://permaplant:permaplant@127.0.0.1/permaplant
 BIND_ADDRESS_HOST=127.0.0.1
 BIND_ADDRESS_PORT=8080
+
+# OAuth2 (other URLs will be fetched from this URL)
+AUTH_DISCOVERY_URI=http://localhost:8081/realms/PermaplanT/.well-known/openid-configuration
+AUTH_CLIENT_ID=PermaplanT
```

Comment:
Why would we want to configure this?

Reply:
There are different clients configured for different instances e.g. https://www.permaplant.net or https://pr.permaplant.net/.
Those need different CLIENT_IDs.

Reply:
I would ask @lukashartl to add that there then. Maybe even write in the doc how the client_id will be called per deployment.

@markus2330 Another option would be to hardcode the client_id (although I don't really like hardcoding values like that) and instead use different realms.
The current solution would mean we can/have to use the same users for https://dev.permaplant.net, https://pr.permaplant.net and https://www.permaplant.net. With different realms users are separate and you could for example only allow us developers on https://pr.permaplant.net. 

Again I would refer to @lukashartl for that as he is the one setting up Keycloak.

Reply:
Yes, I have to agree it would be better to have the same users on all three instances. But what is the purpose of the client_id then? Is it possible to have a test user for only one of the instances?

Reply:
The have different origins. In Keycloak you have to define the root URL and also where you are allowed to redirect users to for each client.
E.g. https://dev.permaplant.net is a different root URL than https://pr.permaplant.net. So if you try to log in via a client that has https://dev.permaplant.net as root URL, but you make that call from https://pr.permaplant.net (you try to reuse a client) Keycloak wouldn't allow that. I think it would technically be possible to define multiple allowed redirect URLs, but that seems very hacky.

Reply:
The documentation also lacks which are valid client IDs for auth.permaplant.net (PermaplanT didn't work for me)

Changelog should contain both that:

- now the config gets fetched from backend
- that auth is active

Reply:
The configuration was still set up for the local Keycloak instance and there exists a client ID PermplanT as described in the setup steps.
I changed the sample to point to the dev instance on auth.permaplant

Reply:
The client IDs needed for development are in the documentation (in the setup steps).
With changelog you mean add it in the next meeting notes? Or do we have an actual changelog? If so I couldn't find it.

Reply:
Yes, the Changelog in each meeting note is our actual changelog. For this week #377